stories/core/Slider.stories.tsx

```diff
@@ -0,0 +1,270 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { useMemo, useCallback, useState } from '@storybook/addons';
+import { Body, Slider } from '../../src/core';
+import { SliderProps } from '../../src/core/Slider/Slider';
+import SvgSmileyHappy from '@itwin/itwinui-icons-react/cjs/icons/SmileyHappy';
+import SvgSmileySad from '@itwin/itwinui-icons-react/cjs/icons/SmileySad';
+
+export default {
+  title: 'Input/Slider',
+  component: Slider,
+} as Meta<SliderProps>;
+
+export const Basic: Story<SliderProps> = (args) => {
+  return <Slider {...args} />;
+};
+
+Basic.args = {
+  disabled: false,
+  values: [50],
+  minLabel: <SvgSmileyHappy />,
+  maxLabel: <SvgSmileySad />,
+  tooltipProps: { style: { userSelect: 'none' } },
```

Comment:
Oh, this is weird if every story needs it.

Wonder if this should be part of the css

Reply:
Maybe.  It is really ugly to have the tooltip text get selected when you are moving the thumb.... which happens quite a bit without the prop in storybook.  It may not be needed if the tooltip actually had room to display above the thumb so it would be out of the way when moving the thumb.

Reply:
I think this is a bigger problem than just tooltips.

If there was a block of text below the slider, it would get selected too